File: JPAMountains/src/main/java/com/skilldistillery/mountains/entities/State.java
package com.skilldistillery.mountains.entities;


import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;


@Entity
public class State {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)

	private int id;
	
	private String abbr;
	
	private String name;

	
	
	
	

	

	

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getAbbr() {
		return abbr;
	}

	public void setAbbr(String abbr) {
		this.abbr = abbr;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

}

File: ngMountains/src/app/models/state.ts
export class State {

    //Field
     id: number;

    abbr: string;

    name: string;

    //Constructor

    constructor(id:number = 1, abbr?:string,name?:string){
        this.id=id;
        this.abbr=abbr;
        this.name=name;
    }
}
